aten/src/ATen/native/IndexingUtils.h

```diff
@@ -48,18 +48,12 @@ static C10_UNUSED std::vector<Tensor> expandTensors(const Tensor & self, IOptTen
   return result;
 }
 
-static C10_UNUSED void checkIndexTensorTypes(IOptTensorListRef indices, bool allow_int=false) {
+static C10_UNUSED void checkIndexTensorTypes(IOptTensorListRef indices) {
```

Comment:
This is only used in 3 places, and all of them have `allow_int` set to True or *should* have it set to True.